omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -294,6 +294,28 @@
                         isValid = false;
                     }
                 }
+                <!--Not to exceed number of booking for a specific return date. Default is 100 clients-->
+               <!-- formatting date String from YYYY-MM-DD to DD-MM-YYYY to pass in the Url-->             
+               var selectedDate = jq('#appointmentDate').find("input[type=hidden]").val(); 
+               var splitDate = selectedDate.split("-");
+               var formattedDate = splitDate[2] + "-" + splitDate[1] + "-" + splitDate[0] ; 
+               var URL = '/openmrs/ws/rest/v1/aijar/publicHoliday/' + formattedDate ;
```

Comment:
@reagan-meant This needs to use the OPENMRS_CONTEXT variable 

We have date format functions in aijar.js

Reply:
Have used the OPENMRS_CONTEXT

The function in aijar,js doesnt give me the format i wanted.....it gives dd/mm/yyyy yet i want dd-mm-yyyy